jenkins/README.md

```diff
@@ -16,17 +16,19 @@ Ping @kubernetes/test-infra-maintainers if you have any questions.
 
 Say you want to add a new job, foo.
 
-1. For e2e ci job, add an entry to [`test-infra/jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml`](jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml).
+1. For e2e ci job, add an entry to [bootstrap-ci.yaml](jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml).
```

Comment:
I'd almost rather see the description of these in a README.md in `jenkins/job-configs/kubernetes-jenkins/`, rather than at the `jenkins/` root.